### DISCUSSION TITLE
Wavy underlines not showing on just one Kitty installation

### DISCUSSION THREAD

**Discussion by benfrain** [kovidgoyal/kitty#5335] [Q&A] [answered]
For reasons I don't understand I have a Kitty on one machine that does not show wavy underlines for spelling issues in Neovim.

I have the same Kitty and Neovim versions and config file on three machines but just one has this issue. Tried different fonts, reinstalled Neovim and Kitty (0.25.2) Any idea what the cause may be?

**Comment by kovidgoyal** [comment, marked as answer]
printf '\e[4:3mtest'

if test has underline, kitty is working fine. Probalem will be elsewhere.

**Reply by benfrain** [reply]
Perfect, thanks, Kitty is fine.

**Reply by benfrain** [reply]
Turns out it was this: https://github.com/kovidgoyal/kitty/issues/3235#issuecomment-758354252

Needed to set it to `xterm-kitty`.

Thanks again for the assistance.
